docs/source/scenarios/citi_bike.rst

```diff
@@ -563,19 +563,19 @@ For the example above, the directory structure should be like:
    |-- ~/.maro
```

Comment:
Could you add a part to introduce the two kinds of usage way of citi bike at the begining of "Data Preparation"? I.e:
- Manually call: maro env data generate ...  The advantage is...
- Data can be automatically downloaded and generated if directly call Env(scenario="citi_bike", topology="...")